cmd/ctr/run_windows.go

```diff
@@ -108,10 +113,20 @@ func newContainer(ctx gocontext.Context, client *containerd.Client, context *cli
 		opts = append(opts, containerd.WithProcessArgs(args...))
 	}
 
+	runtime := context.String("runtime")
+	var platform imageSpecs.Platform
+	if runtime == "io.containerd.runtime.v1.lcow" {
```

Comment:
would be nice to have this string (and the others plugin) defined in a package somewhere so we don't duplicated them. Maybe under `runtime/` since it's already used for the event topics. 

wdyt @crosbymichael ?

Reply:
Well the point is that the runtime names are user ( plugin author ) defined and shouldn't be used on in our codebase.  It does not matter as long as something implements the interfaces.

Reply:
It's just annoy me to have to C/C that string and ensure I don't make a spelling mistake. But sure, it's not the end of the world, I'll get an error on first run. Still not very friendly though.

Reply:
My point was we shouldn't be using it in the first place.

Reply:
hum, true.

@darrenstahlmsft are you planning to have the lcow runtime an updated copy of the current one for wcow?

Can't the same one handle both?

Regardless, maybe adding a `--platform` options to run would be better

Reply:
I could add a `--platform` flag to ctr that defaults to the current platform instead (I alluded to this in the PR description). That approach is more flexible for future runtimes that might support multiple backing platforms.

Reply:
@darrenstahlmsft sounds good.  I'd rather push the burden on the clients with containerd and keep the containerd code ignorant of what runtime is being used.

Reply:
@mlaventure the current plan for the LCOW runtime (working locally, cleaning up and creating PRs now) is to keep it separate to the existing windows runtime, but share code where possible. This simplifies the implementation as the runtime would not need a bunch of conditional code based on platform, and lets clients choose a runtime depending on container type rather than having the runtime decide.

Reply:
fixed with the `--platform` flag